src/Model/AcsControlInterface.php

```diff
@@ -37,13 +37,16 @@ interface AcsControlInterface extends JsonSerializable
     public const ACTION_TERMINATE = 'terminate';
     public const ACTION_UPDATE = 'update';
     public const ACTION_FLAG = 'flag';
+    public const ACTION_DELETE = 'delete';
+    public const ACTION_RESET = 'reset';
```

Comment:
These are not real ACS actions. We must not add them here.

Let's close this PR please.